Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterWindow.cpp

```diff
@@ -917,20 +918,25 @@ namespace EMStudio
 
         const AZStd::string oldName = parameter->GetName();
 
-        // create and init the dialog
-        ParameterCreateEditDialog* dialog = new ParameterCreateEditDialog(m_plugin, this, parameter);
-        dialog->Init();
-        // We cannot use exec here as we need to access it from the tests
-        EMStudio::ParameterCreateEditDialog::connect(dialog, &QDialog::finished, [=](int resultCode)
+        // create and init the widget
+        if (!m_parameterCreateEditWidget)
+        {
+            m_parameterCreateEditWidget = new ParameterCreateEditWidget(m_plugin, this);
+            m_parameterCreateEditWidget->Init(parameter);
+        }
+        else
+        {
+            m_parameterCreateEditWidget->Reinit(parameter);
+            QObject::disconnect(m_parameterCreateEditWidget, &EMStudio::ParameterCreateEditWidget::accept, nullptr, nullptr);
+        }
+
+        EMStudio::InspectorRequestBus::Broadcast(&EMStudio::InspectorRequestBus::Events::Update, m_parameterCreateEditWidget);
+        EMStudio::ParameterCreateEditWidget::connect(m_parameterCreateEditWidget, &EMStudio::ParameterCreateEditWidget::accept, [=]()
```

Comment:
update: Same comment as before about `=` (it also looks like there's a lot of duplicated code here very similar to the earlier section, can any of this be factored out into a reusable call we can use in both places?)